backend/server/models.py

```diff
@@ -14,8 +14,8 @@ class Profile(models.Model):
 
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     nickname = models.CharField(max_length=20)
-    photo = models.ImageField(upload_to=pic_folder, default=DEFAULT_IMAGE)
-    email = models.EmailField(max_length=30)
+    photo = models.ImageField(upload_to=pic_folder, blank=True)
+    # email = models.EmailField(max_length=30)
```

Comment:
@corvuspink  It can be removed, since it will not really useful...

Reply:
I think we still need the email field for social login.

we need some special key that distinguish users, and we can get some information from social (kakao, naver, ...) api. Maybe we can get email address from api.

If we will use username as unique key, and can get username from api or get username from email address, we should remove email field.

And, later on ,maybe for email-validation for blocking malicious registering, we might still need email field.

Reply:
Then how about remove username field and use email field?

Reply:
Removing username field might be OK.

Email field is required, for the reasons stated by @givenone

We can use email address for signing in and etc.